src/Raven.Client/Documents/Session/IDocumentQuery.cs

```diff
@@ -67,6 +67,16 @@ public interface IRawDocumentQuery<T> :
     {
     }
 
+    public interface IGraphQuery<T> :
+        IQueryBase<T, IGraphQuery<T>>,
+        IDocumentQueryBase<T>, IEnumerable<T>
```

Comment:
We should not inherit from `IEnumerable<T>` but I guess this is related to https://issues.hibernatingrhinos.com/issue/RavenDB-10520 and for now we do not have much choice?